src/app/components/sidebar/ActiveLink.tsx

```diff
@@ -0,0 +1,41 @@
+'use client';
+
+import styles from '../../../../styles/app/components/sidebar/nav.module.css';
+import {usePathname} from 'next/navigation';
+import {ReactNode} from 'react';
+import {convertFirstLetterToLowercase} from '../../utils/convertFirstLetterToLowercase';
+
+export const ActiveLink: React.FC<{
+	linkRoute: string;
+	compositionId?: string;
+	formatId?: boolean;
+	className?: string;
+	children: ReactNode;
+}> = ({
+	linkRoute,
+	compositionId,
+	formatId = false,
+	className = '',
+	children,
+}) => {
+	const pathname = usePathname();
+
+	const link = formatId
+		? compositionId && linkRoute + convertFirstLetterToLowercase(compositionId)
+		: linkRoute + compositionId;
```

Comment:
I don't know if it's really necessary to keep the uppercase letter for the showcase route, if so it will gravelly simplify this code, what do you think @CruuzAzul